lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,410 @@
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: true,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        setState(() => _showSettings = !_showSettings);
+                      },
+                      child: const Text('Settings'),
+                    ),
+                    // A button to clear the buffers.
+                    ElevatedButton(
+                      onPressed: _bufferPoints.isEmpty
+                          ? null
+                          : () {
+                              clearBufferPoints();
+                              setState(() => _status = Status.addPoints);
+                            },
+                      child: const Text('Clear'),
+                    ),
+                  ],
+                ),
+                // Conditionally display the settings.
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: const SizedBox.expand(
+                child: ColoredBox(
+                  color: Colors.white30,
+                  child: Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+            // Display a banner with instructions at the top.
+            SafeArea(
+              child: IgnorePointer(
+                child: Container(
+                  padding: const EdgeInsets.all(10),
+                  color: Colors.white.withOpacity(0.7),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Text(
+                        _status.label,
+                        textAlign: TextAlign.center,
+                        style: const TextStyle(color: Colors.black),
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      bottomSheet: _showSettings ? buildSettings(context, setState) : null,
+    );
+  }
+
+  void onMapViewReady() {
+    // Initialize the boundary polygon and the symbols.
+    _boundaryPolygon = _makeBoundaryPolygon();
+    _initializeSymbols();
+
+    // Create a map with the spatial reference as the basemap and add it to our map controller.
```

Comment:
```suggestion
    // Create a map with the defined spatial reference and add it to our map controller.
```
The spatial reference isn't really a basemap here so we can remove that note.

Reply:
Changed the comment.